### ISSUE TITLE
make of MyModule is covered by Pull Request #318: Make NEST installation independent 

### ISSUE THREAD

**Issue by mer0mingian** [nest/nest-simulator#334]
Dear all,

when going through the tutorial for the example module, I get the following error messages after a fresh install (built from 7df53d71601394cbb074d920399466f211e93e16) and registering nest-config when making MyModule:

````
mer0@mer0-Studio-XPS-1340:~/nest/extmodules$ mkdir mmb
mer0@mer0-Studio-XPS-1340:~/nest/extmodules$ cd mmb/
mer0@mer0-Studio-XPS-1340:~/nest/extmodules/mmb$ cmake -Dwith-nest=nest-config ../MyModule
-- The CXX compiler identification is GNU 4.9.2
-- Check for working CXX compiler: /usr/bin/c++
-- Check for working CXX compiler: /usr/bin/c++ -- works
-- Detecting CXX compiler ABI info
-- Detecting CXX compiler ABI info - done

-------------------------------------------------------
mymodule Configuration Summary
-------------------------------------------------------

C++ compiler         : /usr/bin/c++
Build static libs    : OFF
C++ compiler flags   : 
NEST compiler flags  : -O2 -Wall -fopenmp -I/home/mer0/gitdata/nest-dev/{libnestutil,librandom,sli,nestkernel,nest,topology,precise} -I/usr/include -I/usr/include -I/usr/include
NEST libraries flags : -L/home/mer0/opt/nest210dev/lib -lnest_lib -lrandom -lsli_lib -lnestkernel -lnestutil -lprecise -ltopology -fopenmp /usr/lib/x86_64-linux-gnu/libltdl.so /usr/lib/x86_64-linux-gnu/libreadline.so /usr/lib/x86_64-linux-gnu/libncurses.so /usr/lib/libgsl.so /usr/lib/libgslcblas.so

-------------------------------------------------------

You can build and install mymodule now, using
  make
  make install

mymodule will be installed to: /home/mer0/opt/nest210dev/lib/nest

-- Configuring done
-- Generating done
-- Build files have been written to: /home/mer0/nest/extmodules/mmb
mer0@mer0-Studio-XPS-1340:~/nest/extmodules/mmb$ make
Scanning dependencies of target mymodule_lib
[ 25%] Building CXX object CMakeFiles/mymodule_lib.dir/mymodule.cpp.o
In file included from /home/mer0/nest/extmodules/MyModule/mymodule.cpp:23:0:
/home/mer0/nest/extmodules/MyModule/mymodule.h:27:25: fatal error: slifunction.h: No such file or directory
 #include "slifunction.h"
                         ^
compilation terminated.
CMakeFiles/mymodule_lib.dir/build.make:54: recipe for target 'CMakeFiles/mymodule_lib.dir/mymodule.cpp.o' failed
make[2]: *** [CMakeFiles/mymodule_lib.dir/mymodule.cpp.o] Error 1
CMakeFiles/Makefile2:91: recipe for target 'CMakeFiles/mymodule_lib.dir/all' failed
make[1]: *** [CMakeFiles/mymodule_lib.dir/all] Error 2
Makefile:137: recipe for target 'all' failed
make: *** [all] Error 2
mer0@mer0-Studio-XPS-1340:~/nest/extmodules/mmb$ 
```
`

Is "slifunction.h" in mymodule.h intended as a placeholder? If yes, how to adapt it?
I am still new to NEST, so any help would be appreciated.
Thanks in advance!

**Comment by mer0mingian** [comment]
Thanks for the immediate response.
All issues resolved.

**Comment by tammoippen** [comment]
As the title suggests, this is fixed by #318.
